src/components/header/header.js

```diff
@@ -23,7 +23,11 @@ export default class Header {
       header = createElementsFromString(usersHeader(headersProps));
     }
     const screenWithHeader = target.appendChild(header);
-    target.querySelector('.header-content__switch-mode-button').addEventListener('click', () => { switchMode(props); });
+    if (getUserInfo().type == roles.manager) {
```

Comment:
Better use `===` instead of `==`. 
It`s a common practice, also tslint should warn you about that.